common/core/src/retry_operation.ts

```diff
@@ -17,18 +20,20 @@ export class RetryOperation {
   private _maxTimeout: number;
   private _operationStartTime: number;
   private _operationExpiryTime: number;
+  private _name: string = '';
+  private _id: number = 0;
 
   /**
    * Creates an instance of {@link azure-iot-common.RetryOperation.}
    * @param {RetryPolicy} policy The retry policy to be used for this operation, which determines what error is "retryable" or not and how fast to retry.
    * @param {number} maxTimeout  The maximum timeout for this operation, after which no retry will be attempted.
    */
-  constructor (policy: RetryPolicy, maxTimeout: number) {
-    if (policy && policy.constructor && policy.constructor.name === 'NoRetry') {
```

Comment:
We absolutely have to keep this log in the code.  This is a specific request from CSS.